packages/node_modules/@ciscospark/react-component-new-messages-separator/src/styles.css

```diff
@@ -1,8 +1,5 @@
-:root {
-  --color-primary: #049fd9;
-}
-
 .new-messages {
+  --color-primary: #049fd9;
   color: var(--color-primary);
```

Comment:
This seems a big weird that we're declaring a variable and then just setting the color to it. What is the point?